tools/releases/helm.sh

```diff
@@ -100,28 +100,31 @@ function release {
 
   git clone --single-branch --branch "${GH_PAGES_BRANCH}" "$GH_REPO_URL"
 
+  CHART_TAR=$(find ${CHARTS_PACKAGE_PATH}/*.tgz -type f)
+  CHART_FILE=$(tar -tf ${CHART_TAR} | grep 'Chart.yaml')
+  CHART_VERSION=$(tar -zxOf ${CHART_TAR} ${CHART_FILE} | yq .version)
+
+  pushd ${GH_REPO}
+
   # First upload the packaged charts to the release
   cr upload \
     --owner "${GH_OWNER}" \
     --git-repo "${GH_REPO}" \
     --token "${GH_TOKEN}" \
-    --package-path "${CHARTS_PACKAGE_PATH}"
+    --package-path "../${CHARTS_PACKAGE_PATH}"
 
   # Then build and upload the index file to github pages
   cr index \
     --owner "${GH_OWNER}" \
     --git-repo "${GH_REPO}" \
-    --charts-repo "${CHARTS_REPO_URL}" \
```

Comment:
Argument `--charts-repo` had been removed in `cr index`. It is this argument caused issue #13252 : it was previously fetching index from the GH page website instead of the file in the git workspace.